patches/server/0001-Setup-Gradle-project.patch

```diff
@@ -105,6 +105,11 @@ index 0000000000000000000000000000000000000000..71f781083f2f81d064605b2a652181f2
 +}
 +
 +relocation {
++    val packageVersion = "1_17_R1"
```

Comment:
Add a comment that order matters

Reply:
Yeah, moving this back to the bottom was an oversight, adding a comment should avoid that in the future